webpack/assets/javascripts/react_app/stores/NotificationsStore.js

```diff
@@ -12,9 +12,13 @@ class NotificationsEventEmitter extends AppEventEmitter {
   }
 
   getNotifications() {
-    return (_notifications.data || []);
+    return (_notifications.data || {});
   }
 
+  hasUnreadNotifications(notifications) {
```

Comment:
that function I think returns true if any of the notifications has its seen property set to true